Genoa Event Pictures, Minor Burma Update, and Minor Italian Flags

Mr-Parrot opened the pull request.

Xerodoeht commented:
Not sure on removing the burmese homelands in those provinces, the pegu addition was a good catch though. Tenasserim(16%) to me seems more reasonable than kachin, which is 20% burmese. But even then, those two regions are part of burma today, and make sense as burmese homelands, they have a decent amount of burmese.

MadJamez commented:
@Mr-Parrot Can you provide a list with exactly what you changed or added in the description of this pull request?
That will make it easier for me to create the change log. Except for that, this looks good.

Mr-Parrot commented:
Change log,

Releasable: Sicily and Naples 
Formables: Belarus and Corsica-Piedmont
Burma:
- Improved education rates in Burma
- Added Burmese homelands
Changed images of Genoa event changes
Flag: Added flags, Siciliy, Naples,  Etruria, Genoa, Milan, Papal State
Added: 3 clothes factories in Milan and 2 gold mines in Sardinia.
